### DISCUSSION TITLE
Half-life / clearance in ciprofloxacin tutorial

### DISCUSSION THREAD

**Discussion by MarjoleinvB** [Open-Systems-Pharmacology/Forum#1520] [Questions & Problems] [answered]
Hi,

I am trying to learn how to work with PK Sim by the available tutorials. I have a question about the half life output. Ciprofloxacin has a half life of ~4h. But in the IV simulation it is calculated as 7h and for the oral simulation the half life is 11h. 
I have tried to build my own simulations and have often the problem of a too long half life. 
How does this work in PK Sim and is it possible to solve?

![Schermafbeelding 2023-05-02 122805](https://user-images.githubusercontent.com/129178779/235643011-88457afa-27ed-44b2-9323-6b9be278fa53.jpg)

**Comment by tobiasK2001** [comment, marked as answer]
Hi,

Terminal half life time (calculated from the terminal 10% of data points).
(https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations)

That is likely to be different to a half life calculated by e.g. NCA were a line is fitted through the last three data points.

Hope this might helps.

Best, Tobias

**Reply by MarjoleinvB** [reply]
Hi Tobias,

That can indeed explain the difference. If you would like to evaluate how well your models predictions are compared to reality, which PK parameter would you take besides AUC and Cmax. Since it would be helpful to have something like the t1/2 to compare to say something about how well the model describes the clearance. 

Thank you,

Marjolein

**Reply by Atekton** [reply]
Your advice is also helpful to me.
Thanks for your discussion.

Sincerely, Atekton

**Comment by tobiasK2001** [comment]
Dear Marjolein,

when comparing simulated vs observed PK parameters I would go for AUC (inf and tend), Cmax and maybe Tmax as "direct" parameters.
Other parameters like t1/2 or CL or more indirect in the sense as the contain implicit models (e.g. well stirred model) or assumptions (e.g. what is the terminal phase?). I would avoid these and use e.g. GOF plots showing plasma profiles. 
Maybe others have more or different thoughts on that.

Best, Tobias

**Reply by MarjoleinvB** [reply]
Dear Tobias,

Thank you for your advice! I agree that it is better to compare the 'direct' parameters. 

Best, Marjolein
